src/box/lua/upgrade.lua

```diff
@@ -1421,6 +1441,7 @@ local handlers = {
     {version = mkversion(2, 10, 5), func = upgrade_to_2_10_5},
     {version = mkversion(2, 11, 0), func = upgrade_to_2_11_0},
     {version = mkversion(2, 11, 1), func = upgrade_to_2_11_1},
+    {version = mkversion(3, 0, 0), func = upgrade_to_3_0_0},
```

Comment:
I don't see a test that would check that `box.schema.upgrade()` actually works as expected, like [this one](https://github.com/tarantool/tarantool/blob/73e59094b328f870eef31d5155f89177b6e4542b/test/box-luatest/user_auth_history_last_modified_upgrade_test.lua#L63).

Reply:
Done.